pkg/operator/ceph/nfs/config.go

```diff
@@ -48,12 +49,17 @@ func getNFSUserID(nodeID string) string {
 	return fmt.Sprintf("nfs-ganesha.%s", nodeID)
 }
 
-func getNFSClientID(n *cephv1.CephNFS, name string) string {
-	return fmt.Sprintf("client.%s", getNFSUserID(getNFSNodeID(n, name)))
+func getNFSClientID(name string) string {
+	return fmt.Sprintf("client.%s", getNFSUserID(getNFSNodeID(name)))
 }
 
-func getNFSNodeID(n *cephv1.CephNFS, name string) string {
-	return fmt.Sprintf("%s.%s", n.Name, name)
+func getNFSNodeID(name string) string {
```

Comment:
Should we have a new change behind a version check, to support both the older version as well as new version changes?

Reply:
Thats a good point discuss, 
Just  to note we didnt had a check before when we fix other breaking changes https://github.com/rook/rook/pull/15393

Reply:
Hopefully these changes are compatible with different ceph/ganesha versions, and no need for a ceph version check.